internal/admin/dashboard/templates/index.html

```
@@ -152,6 +152,7 @@ <h3>Request Log</h3>
                     <option :value="p" x-text="p"></option>
                 </template>
             </select>
+            <input type="text" placeholder="User path /team/alpha" x-model="usageLogUserPath" @input.debounce.300ms="fetchUsageLog(true)" class="filter-input">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)